[DOCS] Rework semantic search main page (#112452)
elastic/elasticsearch: docs/reference/search/search-your-data/semantic-search-deploy-model.asciidoc

[[semantic-search-deployed-nlp-model]]
=== Tutorial: semantic search with a deployed model

++++
<titleabbrev>Semantic search with deployed model</titleabbrev>
++++

IMPORTANT: For the easiest way to perform semantic search in the {stack}, refer to the <<semantic-search-semantic-text, `semantic_text`>> end-to-end tutorial.

> This guide shows you how to implement semantic search with models deployed in {es}: from selecting an NLP model, to writing queries.

Reviewer: Maybe could have a couple of background sentences that explain that this tutorial was written before the inference endpoints and semantic_text options were introduced and therefore are a little dated. Just one or two extra lines of context might be helpful to re-route the reader.

Maybe that could all live in the IMPORTANT block too?

[[semantic-search-deployed-nlp-model]]
=== Tutorial: semantic search with a deployed model

++++
<titleabbrev>Semantic search with deployed model</titleabbrev>
++++

[IMPORTANT]
====
* For the easiest way to perform semantic search in the {stack}, refer to the <<semantic-search-semantic-text, `semantic_text`>> end-to-end tutorial.

> * This tutorial was written before the {infer} endpoint and `semantic_text` options were introduced. Today we have simpler options for performing semantic search. 

Reviewer 2: ```suggestion
* This tutorial was written before the {infer} endpoint and `semantic_text` field type was introduced. Today we have simpler options for performing semantic search. 
```

Also can we add links to the [inference API](https://www.elastic.co/guide/en/elasticsearch/reference/current/inference-apis.html) and [semantic_text field type](https://www.elastic.co/guide/en/elasticsearch/reference/current/semantic-text.html) documentation here?

> * This tutorial was written before the {infer} endpoint and `semantic_text` options were introduced which makes this as the most complicated option. We recommend to use another method to perform semantic search. 

Reviewer: ```suggestion
* This tutorial was written before the {infer} endpoint and `semantic_text` options were introduced. Today we have simpler options for performing semantic search. 
```
